### PR TITLE
Fixed Incorrect way to use access_token #777

### PR DESCRIPTION
**Pull request by AnupKumarPanwar**
Fixes 
Fixed Incorrect way to use access_token #777 

Changes: 
Removed unnecessary getting of access_token from the request. 

Screenshots for the change: 
NA

### CODE REVIEW

**Comment by AnupKumarPanwar** [comment]
@Akshat-Jain Kindly try to understand the code. Earlier the access_token was being fetched from the get parameters and then it was being used to only check if it's null then throw exception. When it's already mentioned that the base role is user then we need not make such checks. Moreover the code is no where restricting the CMS to send access_token in the URL.

**Comment by Akshat-Jain** [comment]
@AnupKumarPanwar Thanks for updating the sample URL. That was exactly what I was talking about. Thanks :-)

**Comment by akshatnitd** [comment]
@mariobehling @Orbiter Can we have it merged?

File: src/ai/susi/server/api/cms/FiveStarRateSkillService.java [hunk 1]
```diff
@@ -70,17 +70,12 @@ public ServiceResponse serviceImpl(Query call, HttpServletResponse response, Aut
         String skill_name = call.get("skill", null);
         File skill = SusiSkill.getSkillFileInLanguage(language, skill_name, false);
         String skill_stars = call.get("stars", null);
-        String access_token = call.get("access_token", null);
```
Comment (sudheesh001) [inline src/ai/susi/server/api/cms/FiveStarRateSkillService.java]: The relevant string containing the sample query `http://localhost:4000/cms/fiveStarRateSkill.json?model=general&group=Knowledge&skill=aboutsusi&stars=3&access_token=6O7cqoMbzlClxPwg1is31Tz5pjVwo3` needs to be updated in this file now that the access_token is removed.

Comment (AnupKumarPanwar) [reply src/ai/susi/server/api/cms/FiveStarRateSkillService.java]: Sample endpoint remains the same, just the nullity check on access_token in get parameters has been removed.

Comment (AnupKumarPanwar) [reply src/ai/susi/server/api/cms/FiveStarRateSkillService.java]: @sudheesh001 Do you agree with my comment?

Comment (sudheesh001) [reply src/ai/susi/server/api/cms/FiveStarRateSkillService.java]: Agree with this.

**Pull request merged**